resources/lib/service.py

```diff
@@ -20,6 +21,24 @@ def __init__(self):
             if self.waitForAbort(10):
                 break
 
+    @staticmethod
+    def onNotification(sender, method, data):  # pylint: disable=invalid-name
+        ''' Handler for notifications '''
+        _kodi = KodiWrapper(None)
+
+        if 'upnextprovider' in sender:
+            import json
+            hexdata = json.loads(data)
+
+            if hexdata:
+                from binascii import unhexlify
+                data = json.loads(unhexlify(hexdata[0]))
+                _kodi.log('[Up Next]: %s, %s, %s' % (sender, method, to_unicode(data)), 'Verbose')
+
+                if 'plugin.video.vrt.nu_play_action' in method:
+                    executebuiltin('PlayerControl(Stop)')
+                    executebuiltin('PlayMedia(plugin://plugin.video.vrt.nu/play/whatson/%s)' % data.get('whatson_id'))
```

Comment:
And I guess this could become:
```python
                utils.jsonrpc(method='Addons.ExecuteAddon',  params=dict(addonid='plugin.video.vrt.nu', params=['/play/whatson/%(whatson_id)s/' % data]))
```

Reply:
I tried various things, none of these would work:
```python
        _kodi.jsonrpc(method='Player.Stop', params=dict(playerid=_kodi.get_playerid()))
        _kodi.jsonrpc(method='GUI.ActivateWindow',  params=dict(
            window='videos',
            params=['plugin://plugin.video.vrt.nu/play/whatson/%(whatson_id)s/' % data],
        ))
```
```python
        _kodi.jsonrpc(method='Player.Stop', params=dict(playerid=_kodi.get_playerid()))
        _kodi.jsonrpc(method='Addons.ExecuteAddon',  params=dict(
            addonid='plugin.video.vrt.nu',
            params=['/play/whatson/%(whatson_id)s/' % data],
        ))
```
```python
        _kodi.jsonrpc(method='Player.Stop', params=dict(playerid=_kodi.get_playerid()))
        _kodi.jsonrpc(method='Player.Open', params=dict(item=dict(episodeid=data.get('whatson_id'))))
```

Reply:
I tried
```python
        _kodi.jsonrpc(method='Player.Open', params=dict(item=dict(file='plugin://plugin.video.vrt.nu/play/whatson/%s' % data.get('whatson_id'))))
```
and it worked.